refresh TopSQL docs and screenshots (#22559)
pingcap/docs: tikv-configuration-file.md

+ In a default TSO physical time update interval (`50ms`), PD provides at most 262144 TSOs. When requested TSOs exceed this number, PD provides no more TSOs. This configuration item is used to avoid exhausting TSOs and the reverse impact of TSO exhaustion on other businesses. If you increase the value of this configuration item to improve high availability, you need to decrease the value of [`tso-update-physical-interval`](/pd-configuration-file.md#tso-update-physical-interval) at the same time to get enough TSOs.
+ Default value: `8192`

## resource-metering

Configuration items related to resource metering.

### `enable-network-io-collection` <span class="version-mark">New in v8.5.6 and v9.0.0</span>

> + Whether to additionally collect TiKV network traffic and logical I/O information in [Top SQL](/dashboard/top-sql.md).

Reviewer: ```suggestion
+ Whether to collect TiKV network traffic and logical I/O information in [Top SQL](/dashboard/top-sql.md) in addition to CPU data.
```

+ When enabled, TiKV additionally records inbound network bytes, outbound network bytes, logical read bytes, and logical write bytes during request processing.
+ When reporting resource consumption, TiKV filters the Top N records based on CPU time, network traffic, and logical I/O, and additionally reports these statistics by Region for more fine-grained analysis of hotspot requests or resource usage sources.
+ Default value: `false`

> **Note:**
>

> > Logical I/O refers to the logical amount of data processed by requests at the TiKV storage layer, such as the amount of data scanned or processed during reads, and the logical write bytes of write requests themselves. Physical I/O refers to the actual disk read/write traffic that occurs on the underlying storage device, which is affected by block cache, compaction, flush, and other factors. Therefore, logical I/O is not equivalent to actual physical I/O, and they cannot be directly mapped one-to-one.

Reviewer: ```suggestion
> Logical I/O is not equivalent to physical I/O and cannot be directly correlated:
>
> - Logical I/O refers to the logical amount of data processed by requests at the TiKV storage layer, such as data scanned or processed during reads and data written by write requests.
> - Physical I/O refers to the actual disk read/write traffic on the underlying storage device, which is affected by block cache, compaction, flush, and other factors.
```

+ In a default TSO physical time update interval (`50ms`), PD provides at most 262144 TSOs. When requested TSOs exceed this number, PD provides no more TSOs. This configuration item is used to avoid exhausting TSOs and the reverse impact of TSO exhaustion on other businesses. If you increase the value of this configuration item to improve high availability, you need to decrease the value of [`tso-update-physical-interval`](/pd-configuration-file.md#tso-update-physical-interval) at the same time to get enough TSOs.
+ Default value: `8192`

## resource-metering

Configuration items related to resource metering.

### `enable-network-io-collection` <span class="version-mark">New in v8.5.6 and v9.0.0</span>

> + Whether to collect TiKV network traffic and logical I/O information in [Top SQL](/dashboard/top-sql.md) in addition to CPU data.

Reviewer 2: ```suggestion
+ Controls whether to collect TiKV network traffic and logical I/O information in [Top SQL](/dashboard/top-sql.md) in addition to CPU data.
```
